src/plots/plots.js

```diff
@@ -2886,13 +2885,25 @@ function sortAxisCategoriesByValue(axList, gd) {
         }
     }
 
+    var aggFn = {
+        'min': function(values) {return Lib.aggNums(Math.min, null, values);},
+        'max': function(values) {return Lib.aggNums(Math.max, null, values);},
+        'sum': function(values) {return Lib.aggNums(function(a, b) { return a + b;}, null, values);},
+        'value': function(values) {return Lib.aggNums(function(a, b) { return a + b;}, null, values);},
+        'mean': function(values) {return Lib.mean(values);},
+        'median': function(values) {values.sort(); var mid = Math.round((values.length - 1) / 2); return values[mid];}
```

Comment:
I'm surprised our linter didn't complain here. Would you mind rewriting this as

```js
'median': function(values) {
    values.sort(); 
    var mid = Math.round((values.length - 1) / 2); 
    return values[mid];
}
```

Reply:
There's probably an eslint [rule](https://eslint.org/docs/rules/) for this :microscope: